From 8785c7ebd16b0bce60375a7344d3caf22aa7439a Mon Sep 17 00:00:00 2001
From: Zhang Ze <zhangze@rvcore.com>
Date: Fri, 20 Jan 2023 12:15:26 +0800
Subject: [PATCH] fix issue that g++ can not find header

fix issue that g++ can not find header when compiling verilator v4.218
---
 src/config_build.h.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/config_build.h.in b/src/config_build.h.in
index d044c1411..05f6daca5 100644
--- a/src/config_build.h.in
+++ b/src/config_build.h.in
@@ -67,6 +67,7 @@
 #include <cstring>
 #include <string>
 #include <iostream>
+#include <memory>
 
 // Avoid needing std:: prefixes on some very common items
 using string = std::string;
